scripts/build.sh

```diff
@@ -43,9 +43,8 @@ alias mint_rpc_client="\$MINT_RPC_CLIENT"
 
 # Function for killing processes stored in PID_FILE
 function kill_minimint_processes {
-  while read pid; do
-    kill $pid || true
-  done <$PID_FILE
+  kill $(cat $PID_FILE | sed '1!G;h;$!d') #sed reverses the order here
+  kill $(pidof "ln_gateway")
```

Comment:
I assume that could be a call to `pkill`? Also, doesn't `lightningd` take care of its plugins?